Resources/Prototypes/Guidebook/chemicals.yml

```diff
@@ -0,0 +1,60 @@
+- type: guideEntry
+  id: Chemicals
+  name: guide-entry-chemicals
+  text: "/ServerInfo/Guidebook/Chemicals.xml"
+  children:
+  - Elements
+  - Medicine
+  - Narcotics
+  - Pyrotechnic
+  - Toxins
+  - Foods
+  - Botanical
+  - Biological
+  - Others
+  filterEnabled: True
+
+- type: guideEntry
+  id: Elements
+  name: guide-entry-elements
+  text: "/ServerInfo/Guidebook/Chemical Tabs/Elements.xml"
+
+- type: guideEntry
+  id: Medicine
```

Comment:
CI is failing because this category is already defined under the jobs -> medical tab. Distinguish it with a different ID.

Reply:
I just have to change the ID, right?

Reply:
All checks green!